src/cli/commands/telemetry/register.ts

```diff
@@ -0,0 +1,17 @@
+import { Command } from "commander";
+import { configureOptions } from "../../../core";
+import { telemetry } from "./telemetry";
+
+export default function registerCommand(program: Command) {
+  program
+    .command("telemetry")
+    .usage("[options]")
+    .description("enable/disable SWA CLI telemetry capturing")
+    .option("-dt, --disable", "opt out of telemetry", undefined)
```

Comment:
any reason for using `-dt` two characters in the option? why not we are using `-d`, `-e` etc here?

Reply:
-d, -e and -s are already in use for other swa commands

Reply:
I see so these short forms are not within a command group its global